quixstreams/sources/kafka/quix.py

```diff
@@ -0,0 +1,110 @@
+from typing import Optional, TYPE_CHECKING
+
+from quixstreams.error_callbacks import default_on_consumer_error, ConsumerErrorCallback
+from quixstreams.kafka import AutoOffsetReset
+from quixstreams.models.serializers import DeserializerType
+from quixstreams.platforms.quix import QuixKafkaConfigsBuilder
+from quixstreams.platforms.quix.api import QuixPortalApiService
+
+from .kafka import KafkaSource
+
+if TYPE_CHECKING:
+    from quixstreams.app import ApplicationConfig
+
+
+__all__ = ["QuixEnvironmentSource"]
+
+
+class QuixEnvironmentSource(KafkaSource):
+    """
+    Source implementation that replicate a topic from a quix workspace to your application cluster.
```

Comment:
```suggestion
    A Source implementation that replicates a topic from an environment in Quix Cloud to your application cluster.  
    
    It can copy messages for development and testing without risking producing them back or affecting the consumer groups.
```